src/gui/mzroll/spectrawidget.cpp

```diff
@@ -885,7 +885,7 @@ void SpectraWidget::addAxes()
     	Axes* y = new Axes(1, _minY, _maxY, 10);
     	scene()->addItem(y);
         y->setZValue(999);
-        y->showTicLines(false);
+        y->showTickLines(false);
```

Comment:
possible typo

Reply:
I renamed the `Axes` method appropriately, so this is intentional.